packages/quantic/force-app/main/default/lwc/quanticFoldedResultList/quanticFoldedResultList.html

```diff
@@ -8,7 +8,7 @@
       <c-quantic-placeholder variant="resultList" number-of-rows={numberOfResults}></c-quantic-placeholder>
     </template>
     <template for:each={collections} for:item="collection">
-      <c-quantic-result key={collection.result.uniqueId} result={collection.result} collection={collection}
+      <c-quantic-result key={collection.keyResultList} result={collection.result} collection={collection}
```

Comment:
Just a naming suggestion if you decide to go with it.
```suggestion
      <c-quantic-result key={collection.resultKey} result={collection.result} collection={collection}
```